test/keyv.js

```diff
@@ -85,6 +85,14 @@ test.serial('.get(key, {raw: true}) returns the raw object instead of the value'
 	t.is(rawObject.value, 'bar');
 });
 
+test.serial('.has(key) where key is the key we are looking for', async t => {
```

Comment:
This tests should actually be in [`keyv-test-suite`](https://github.com/lukechilds/keyv-test-suite/) so they are run against all storage adapters and catch any anomalies.

Don't worry about that for now, you can add the tests here and I'll move them over. The development process is a bit painful because all the modules depend on each other.

Would you be able to make the tests a bit more extensive though, it should test all possible different types of behaviour with all types. Ideally a single test for each check.

It may seem over the top and it's fine with the current implementation because you know it'll always return an object or undefined. But that's testing the current implementation, not the functionality. If in the future we made a change to how entries are stored internally it could mean that a value of undefined would then cause `.has()` to return false when it should return true.

You should extensively test all expected behaviour with all types.

You can use these test files for inspiration:
https://github.com/lukechilds/keyv-test-suite/blob/master/src/api.js
https://github.com/lukechilds/keyv-test-suite/blob/master/src/values.js